lib/elixir/pages/getting-started/pattern-matching.md

```diff
@@ -122,9 +122,10 @@ iex> x = 1
 iex> x = 2
 2
 ```
+
 However, there are times when we don't want variables to be rebound.
 
-Use the pin operator `^` when you want to pattern match against a variable's _existing value_ rather than rebinding the variable.
+Use the pin operator `^` when you want to pattern match against a variable's *existing value* rather than rebinding the variable.
```

Comment:
I am almost sure we use `_` for italics and `**` for bold throughout the docs. Am I mistaken?

Reply:
I think so, yes. If I search for the regex `\*\w+\*` I get a ton of results, while scanning the results for `_\w+_` I can't see any in the docs, they're all in code.